internal/commands/commands.go

```diff
@@ -177,4 +178,19 @@ var (
 			},
 		},
 	}
+
+	AccessList = cli.CommandDefinition{
+		CommandMeta: cli.CommandMeta{
+			Use:         "accessList",
+			Aliases:     []string{"accesslist", "access-list"},
+			Description: "Manage your allowed IP addresses and CIDR blocks",
+			//Hidden:      true, // TODO(REALMC-9164): Remove 'Hidden'
```

Comment:
related to my other comment about `Hidden`, but by not commenting this out we can now hide all `accesslist` commands in one spot, and not worry about setting `Hidden` individually on each subcommand